packages/web/src/components/collection/desktop/OwnerActionButtons.tsx

```diff
@@ -46,7 +46,7 @@ export const OwnerActionButtons = (props: OwnerActionButtonProps) => {
         />
       ) : null}
 
-      {!is_private ? (
+      {is_private ? (
```

Comment:
was this a bug?

Reply:
no great find, was testing different flows